includes/post-types.php

```diff
@@ -76,7 +76,7 @@ function give_setup_post_types() {
 
 	$give_forms_args = array(
 		'labels'             => $give_forms_labels,
-		'public'             => true,
+		'public'             => $give_forms_singular,
 		'publicly_queryable' => $give_forms_singular,
```

Comment:
@ravinderk Yes, it will work. I have not removed it considering that the value of `public` parameter is overriding some other parameters like `exclude_from_search`, `publicly_queryable`, `show_ui`, and `show_in_nav_menus`. So, i thought to keep it as it is considering that it can affect when Form Single Views are enabled.